src/js/auth/containers/AuthApp.jsx

```diff
@@ -112,8 +112,9 @@ const mapDispatchToProps = (dispatch) => {
     onUpdateLoggedInStatus: (update) => {
       dispatch(updateLoggedInStatus(update));
     },
+    // Is this ever used?
     onUpdateProfile: (field, update) => {
```

Comment:
I guess I didn't really need to keep the comment, but...is this ever used? It's the only instance of `onUpdateProfile` in the file and it's not being exported or anything, so...

Still, it seemed a bit out of scope to nix it in this PR.